tests/e2e/e2e_test.go

```diff
@@ -390,7 +400,7 @@ var _ = ginkgo.Describe("[Start/Remove/Restart/Add/Stop]", func() {
 
 			v, ok := resp.ClusterInfo.AttachedPeerInfos["node1"]
 			gomega.Ω(ok).Should(gomega.BeTrue())
-			color.Outf("{{green}}successfully attached peer:{{/}} %+v\n", v.Peers)
+			log.Info(logging.Green.Wrap("successfully attached peer: %+v"), v.Peers)
```

Comment:
is this string slice?

Reply:
No this is a `Peers []*AttachedPeerInfo`